### PR TITLE
Add texlive-docker method

### PR DESCRIPTION
**Pull request by PHPirates**
@Strauman I have taken the freedom to add your method, would you want to quickly review it?

The [tex.stackexchange.com](https://tex.stackexchange.com/a/450256/98850) answer should also definitely be updated, preferably containing a complete but minimal setup including a full `.travis.yml` file. Do you want to do that or can I edit it? (not sure the edit would be approved if you're not the one reviewing).

### CODE REVIEW

**Comment by Strauman** [comment]
I have updated my answer there. Do you want me to update yours too? (I have rep so an edit would just be applyed without approval, that's why I'm asking first)

Reviewing now.

**Edit/addendum**
> Do you want to do that or can I edit it? (not sure the edit would be approved if you're not the one reviewing).

If you want, you can just go ahead and edit it. If it doesn't get approved in review, I can override it.

**Comment by Strauman** [comment]
Also - for cool factor - check out the github [code owner](https://blog.github.com/2017-07-06-introducing-code-owners/) feature. If you made me a code owner of the files or directory that contains only "my" stuff, then I could actually Github review this request!

**Edit** Turns out I can github review it anyway!

**Comment by Strauman** [comment]
> Do you want to do that or can I edit it? (not sure the edit would be approved if you're not the one reviewing).

If you want, you can just go ahead and edit it. If it doesn't get approved in review, I can override it.

File: 3-texlive-docker/.travis.yml
Comment (Strauman) [inline 3-texlive-docker/.travis.yml:22]: What are these `file` and `tags` things?

Comment (Strauman) [reply 3-texlive-docker/.travis.yml:22]: Also - couldn't find docs for the `deploy` keys from a quick google? Can you provide a link?

**Comment by PHPirates** [comment]
Great! That was fast. Yeah sure go ahead, do you want to add the complete `.travis.yml` or just a link to your method? The `.travis.yml` is rather short (at least a minimal example) so it should not make the answer much longer. I prefer to keep the answer self-contained and not have too much links.

Aha, you have some mighty privileges there :smile: 

Good one, actually I was already using a code owners file but forgot about it. I hope it will work for a next PR. Indeed, you can review any PR.

File: README.md
Comment (Strauman) [inline README.md:70]: I'll add this as an issue, so it's even easier to change.

File: README.md
Comment (Strauman) [inline README.md:80]: Do you know how to do cache the packages?

File: README.md
Comment (Strauman) [inline README.md:164]: Not yet supported though?

**Review by Strauman — APPROVED** [review]
Looks great!

**Comment by Strauman** [comment]
> do you want to add the complete .travis.yml or just a link to your method?

I'll change the answer to be more self contained when I have time. I don't want to spend 15-20 minutes now and only ending up with rewriting everything at a different time. As I said, if you feel the spirit, just go for it!

File: 3-texlive-docker/.travis.yml
Comment (PHPirates) [reply 3-texlive-docker/.travis.yml:22]: They are for these instructions to upload to GH releases, which I linked to in the last step: https://github.com/PHPirates/travis-ci-latex-pdf#to-automatically-deploy-pdfs-to-github-release
You're right, I forgot to link to the docs! Fixing that.
If I recall correctly I tested that [it worked](https://github.com/PHPirates/travis-ci-latex-pdf/releases/tag/test-v23), like it should since the only thing it uses is that there is a pdf in the right location.

File: README.md
Comment (PHPirates) [reply README.md:70]: Yes I plan to test this, see #12, in this case I would try to compile with lualatex - should be as easy as adding the `-lualatex` flag as option, I hope. I already tried lualatex using the other texlive method, just have to add it to the docs.

File: README.md
Comment (PHPirates) [reply README.md:80]: Nope, it's on my list but I'll create an issue for that. I think we have to commit the docker image, cache it, then reuse it the next time? I think that's possible.
#20

File: README.md
Comment (PHPirates) [reply README.md:164]: As I said, I hope it still works. Maybe you can test it?

**Comment by PHPirates** [comment]
>I don't want to spend 15-20 minutes now and only ending up with rewriting everything at a different time. 

True, DRY :smile: but you know more than me how much you still want to change. I'll add a minimal example and point to it, just to have something. I will remember to update when necessary.

**Pull request merged**
